<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sayı 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Sayı 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSayi1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSayi2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>191</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTopla">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toplama</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCikar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çıkarma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBol">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bölme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCarp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Çarpma</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sonuc">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sonuç </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>293</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
